yoticlient.go

```diff
@@ -99,23 +94,25 @@ func (client *Client) GetSdkID() string {
 
 // GetActivityDetails requests information about a Yoti user using the one time use token generated by the Yoti login process.
 // It returns the outcome of the request. If the request was successful it will include the users details, otherwise
-// it will specify a reason the request failed.
-func (client *Client) GetActivityDetails(token string) (ActivityDetails, []string) {
-	_, activity, errStrings := client.getActivityDetails(token)
-	return activity, errStrings
+// an error it will be returned which will specify the reason the request failed,
+// if the function call can be reattempted with the same token the error will
```

Comment:
Just need to do this change to if you decide to accept my last suggestion of adding a full stop.
```suggestion
// If the function call can be reattempted with the same token the error will
```